services-manager/src/main/java/uk/gov/gchq/palisade/service/manager/config/ServiceConfiguration.java

```diff
@@ -177,13 +158,13 @@ public int hashCode() {
     @Override
     @Generated
     public String toString() {
-        return new StringJoiner(", ", ServiceConfiguration.class.getSimpleName() + "[", "]")
-                .add("jar='" + jar + "'")
-                .add("paths=" + paths)
-                .add("profiles=" + profiles)
-                .add("log=" + log)
-                .add("err=" + err)
-                .add("level=" + level)
+        return new StringJoiner(", ", ServiceConfiguration.class.getSimpleName() + "[", "\n]")
```

Comment:
```suggestion
        return new StringJoiner(", ", ServiceConfiguration.class.getSimpleName() + "[", "]")
```
Do we really need the new line seeing as there is one in the add below?

Reply:
Yeah, this is on the last line. The output then looks like:
```
ServiceConfiguration[
    someString='this',
    someNestedObject=NestedObject[
        someInt=1,
    ],
]
```

It's only nicer here because it is intentionally intended for 'The End User' through use of the --manager.mode=config which prints all this information for debugging in a user-friendly manner

Reply:
Ah, I see. That makes sense.